docs/microbit_micropython_api.rst

```diff
@@ -105,6 +139,34 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # returns boolean
     pin.is_touched()
 
+Except in the case of the pins marked **V2**, which offers the following API::
+    
+    **pin_logo**
+
+    # returns boolean
+    pin_logo.is_touched()
+
+    **pin_speaker**
+    
+    # disable the built-in speaker
+    pin_speaker.disable()
+    # enable the built-in speaker
+    pin_speaker.enable()
+    # value can be 0, 1, False, True
+    pin.write_digital(value)
+    # returns either 1 or 0
+    pin.read_digital()
+    # value is between 0 and 1023
+    pin.write_analog(value)
+    # returns an integer between 0 and 1023
+    pin.read_analog()
+    # sets the period of the PWM output of the pin in milliseconds
+    # (see https://en.wikipedia.org/wiki/Pulse-width_modulation)
+    pin.set_analog_period(int)
+    # sets the period of the PWM output of the pin in microseconds
+    # (see https://en.wikipedia.org/wiki/Pulse-width_modulation)
+    pin.set_analog_period_microseconds(int)
```

Comment:
We could replace `pin` for `pin_speaker`, or remove this and mention that is the same as the rest of the pins (minus touch).

Reply:
Done